salt/metalk8s/defaults.yaml

```diff
@@ -105,4 +105,11 @@ coredns:
 
 upgrade: false        # define if we're on an upgrade case
 
+# Define if the downgrade from this version is supported.
+# It should be set to false if manual operations are needed
+# (e.g. downgrade of etcd), prior to downgrading the cluster.
+# The downgrade can still be forced setting force_downgrade to True
+# in the pillar.
+downgrade_supported: false
```

Comment:
Do we really want a pillar key to say downgrade is supported ? Because it's supported from 2.6.1 to 2.6.0 for example but will never be supported from 2.6.x to 2.5.x ... so to me we should just have a check about the version in the precheck

Reply:
Good point